packages/grid/x-data-grid/src/tests/columns.DataGrid.test.tsx

```diff
@@ -15,7 +15,7 @@ const rows: GridRowsProp = [{ id: 1, idBis: 1 }];
 const columns: GridColumns = [{ field: 'id' }, { field: 'idBis' }];
 
 describe('<DataGridPro /> - Columns', () => {
```

Comment:
As mentioned previously, instead of `columns.DataGrid.test.tsx`, `columnHeaders.DataGrid.test.tsx` is a more suitable place for such tests as the menu part belongs to column headers.

Reply:
🤦‍♂️ Sorry again. Fixed right away.